spec/integration/jara_spec.rb

```diff
@@ -46,20 +38,35 @@ def setup_git(project_dir)
     isolated_run(project_dir,
       'git init --bare ../repo.git',
       'git init',
-      'git add . && git commit -m "Fist!"',
+      'git add .',
+      'git commit -m "First!"',
       'git remote add origin ../repo.git',
       'git push -u origin master'
     )
   end
 
+  def bundle_env(dir)
+    {
+      'BUNDLE_GEMFILE' => File.join(dir, 'Gemfile'),
+      'BUNDLE_PATH' => Dir[File.join(VENDOR_PATH, File.basename(dir), '*', '*')].sort.last,
+    }.map { |k, v| "#{k}=#{v}" }.join(' ')
```

Comment:
When I tested this in an directory with spaces I noticed that the `Rakefile` had the same issue, so fixed it there as well.